### DISCUSSION TITLE
Option to reopen editor later

### DISCUSSION THREAD

**Discussion by FrederriceH** [flameshot-org/flameshot#2135] [Ideas]
Hey Team,

It would be really great to be able to reopen the saved captures later to re-edit them. Like, maybe be able to reopen them from the pin?

An example of when this would be useful would be in taking documentation. I can grab a bunch of screenshots as I go, and then go back and edit, add numbers and comments, blur details etc after capturing all of the screens.

I think it would be a super useful feature!

Thanks! :))

**Comment by mmahmoudian** [comment]
Basically your suggestion is a duplicate of #1100 with some hints of #954. Please feel free to to read those two. Additionally you can checkout #1515 which might be related to your idea.
